internal/ctrfactory/ctrfactory.go

```diff
@@ -1,4 +1,4 @@
-package container
+package ctrfactory
```

Comment:
(GH can't let me comment on the commit so I'm commenting here)

The commit message should be self-explanatory. `ctrfactory: do the same as sboxfactory` is not.

I'd say something like
```
pkg/container: rename to ctrfactory

For reasons behind this, see earlier commit titled "XXXX"
```

or just copy-paste the reasoning as it's not too long.